Assets/MixedRealityToolkit.Examples/Demos/HandTracking/Script/HandInteractionPan.cs

```diff
@@ -489,51 +491,37 @@ public void OnTouchCompleted(HandTrackingInputEventData eventData)
         {
             if (touchingSource == eventData.InputSource)
             {
-                currentPointer = null;
-                currentController = null;
-                touchingSource = null;
-                touchingPoint = Vector3.zero;
-                touchingInitialPt = Vector3.zero;
-                touchingUV = Vector2.zero;
-                touchingInitialUV = Vector2.zero;
-                touchingUVOffset = Vector2.zero;
-                EndTouch();
+                DisconnectTouch();
                 eventData.Use();
             }
         }
     }
-    public void OnTouchUpdated(HandTrackingInputEventData eventData)
-    {
-    }
+    public void OnTouchUpdated(HandTrackingInputEventData eventData) { }
     #endregion IMixedRealityHandTrackHandler
 
 
     #region BaseFocusHandler Methods
-    public override void OnFocusEnter(FocusEventData eventData)
-    {
-    }
+    public override void OnFocusEnter(FocusEventData eventData){}
     public override void OnFocusExit(FocusEventData eventData)
     {
-        if (currentPointer == eventData.Pointer)
-        {
-            DisconnectTouch();
-        }
+        DisconnectTouch();
     }
     #endregion
 
 
     #region IMixedRealityInputHandler Methods
-    public void OnInputDown(InputEventData eventData)
+    public void OnInputDown(InputEventData eventData)                          
     {
-        if (touchType == TouchType.HandRay)
+         if (touchType == TouchType.HandRay)
         {
             if (touchingSource == null)
             {
                 currentPointer = eventData.InputSource.Pointers[0];
                 currentController = currentPointer.Controller;
                 touchingSource = eventData.InputSource;
-                // touchingPoint = currentPointer.Rays[touchingSource.Pointers[0].Result.RayStepIndex].Terminus;
+
                 TryGetHandRayPoint(currentController, out touchingPoint);
```

Comment:
>TryGetHandRayPoint(currentController, out touchingPoint); [](start = 16, length = 57)

Is TryGetHandRayPoint needed any more? It looks like this code is doing something like point = event.pointer.controller.pointer. Seems like you could just do `touchingPoint = eventData.Pointer.Position`.

Reply:
yes- change made